data/styles/vehicle/include/priorities_4_overlays.prio.txt

```diff
@@ -174,7 +174,7 @@ amenity-charging_station-motorcar                   # icon z14- (also has captio
 amenity-fuel                                        # icon z12- (also has caption(optional) z12-)
 === 3200
 
-highway-traffic_signals                             # icon z19-
+highway-traffic_signals                             # icon z15-
```

Comment:
Higher than subway and railway stations even? Hmm..

Reply:
Fixed